presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -83,9 +88,14 @@ public URI getHttpUri()
         return buildHttpUri();
     }
 
+    public Properties getConnectionProperties()
```

Comment:
Maybe you can move the `HttpClientConf` creation here. And call this method `getHttpClientConfiguration`